.gitignore

```diff
@@ -9,3 +9,6 @@
 coverage.txt
 profile.out
 /.glide
+
+# files generated during tests
+config/temp
```

Comment:
Thanks for the info!
That's odd 😭
Would you mind reverting this and then raising an another issue specif for that?